sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Custom/CassandraClusterResource.cs

```diff
@@ -19,42 +19,5 @@ namespace Azure.ResourceManager.CosmosDB
     /// </summary>
     public partial class CassandraClusterResource : ArmResource
```

Comment:
there is no method in this class, can we just remove this file?

Reply:
removed